docs/registry.md

```diff
@@ -1,20 +1,27 @@
 # Configure Image Registry
 This document describes the method to configure the image registry for `containerd` for use with the `cri` plugin.
 
+NOTE: The configuration syntax used in this doc is in version 2 which is the
+default since containerd 1.13.0. If your configuration is still in version 1,
```

Comment:
"default since containerd 1.13.0" ->  "recommended since containerd 1.3" ?

Reply:
fixed, you're right, the default version (if not set) is still the version 1.